launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -214,7 +214,16 @@ QString MinecraftInstance::binRoot() const
 QString MinecraftInstance::getNativePath() const
 {
 #ifdef Q_OS_FREEBSD
-    QDir natives_dir("/usr/local/lib/lwjgl/");
+    QDir natives_dir;
```

Comment:
@txtsd why this?

Reply:
It fails because the return statement is not in the same scope.

```
/home/txtsd/git/polymc/PolyMC/launcher/minecraft/MinecraftInstance.cpp:226:12: error: ‘natives_dir’ was not declared in this scope
  226 |     return natives_dir.absolutePath();
      |            ^~~~~~~~~~~
```